core/src/test/java/io/questdb/griffin/AbstractGriffinTest.java

```diff
@@ -290,7 +287,11 @@ private static void testSymbolAPI(RecordMetadata metadata, RecordCursor cursor)
                     if (symbolTable instanceof StaticSymbolTable) {
                         CharSequence sym = record.getSym(column);
                         int value = record.getInt(column);
-                        Assert.assertEquals(value, ((StaticSymbolTable) symbolTable).keyOf(sym));
+                        if (symbolTable instanceof SymbolMapReader && ((SymbolMapReader) symbolTable).containsNullValue() && value == ((StaticSymbolTable) symbolTable).size()) {
```

Comment:
I don't follow this condition. Why are we comparing int value of symbol to size of symbol map?

Reply:
the reason is that the symboMapReader.size() still returns the number of symbols without counting the null value. on the other hand, record.hasNext() now returns true one final time if the symbolMapReader contains a null symbol

Reply:
perfect, understood, thanks!